src/ductile/document.clj

```diff
@@ -396,10 +400,8 @@
 
 (s/defn delete-by-query-uri
   [uri index-names]
-  (let [index (string/join "," index-names)]
-    (str (uri/uri uri
-                  (uri/uri-encode index)
-                  "_delete_by_query"))))
+  (let [index (uri/uri-encode (string/join "," index-names))]
+    (str (uri/uri uri index "_delete_by_query"))))
```

Comment:
If you change it, you need to add a test case.

Reply:
Simple re-structure, so both delete-by-query-uri and update-by-query-uri look similar. Semantics remain the same. It looks like the function backed up by tests already. See [ductile.document-test/delete-by-query-uri-test](https://github.com/threatgrid/ductile/blob/master/test/ductile/document_test.clj#L26)